client/my-sites/media-library/proxied-image.jsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+
+const { Blob } = globalThis; // The linter complains if I don't do this...?
+
+export default function( { alt, mediaUrl, onLoad = noop, siteSlug, style } ) {
```

Comment:
Naming this function is the difference between seeing `Anonymous` and `ProxiedImage` in dev tools:

<img width="537" alt="Zrzut ekranu 2020-02-24 o 12 38 41" src="https://user-images.githubusercontent.com/205419/75149582-aa7b9b80-5702-11ea-9d81-faaeac1553c7.png">

<img width="495" alt="Zrzut ekranu 2020-02-24 o 12 39 30" src="https://user-images.githubusercontent.com/205419/75149598-b7988a80-5702-11ea-80a7-11f093115af2.png">

```suggestion
export default function ProxiedImage( { alt, mediaUrl, onLoad = noop, siteSlug, style } ) {
```